app/backend/src/couchers/servicers/api.py

```diff
@@ -840,8 +836,8 @@ def user_model_to_pb(db_user, session, context):
             api_pb2.LanguageAbility(code=ability.language_code, fluency=fluency2api[ability.fluency])
             for ability in db_user.language_abilities
         ],
-        regions_visited=[region.region_code for region in db_user.regions_visited],
-        regions_lived=[region.region_code for region in db_user.regions_lived],
+        regions_visited=[region.code for region in db_user.regions_visited],
```

Comment:
Isn't it `region_code` in models?

Reply:
I changed the relationship. They're now relationships between `Region` and `User`, using `RegionsLived` and `RegionsVisited` as secondary tables (which, I think, is how it's supposed to be architecturally). So these `region.code`s are from the `Region` table instead of one of the other `Regions<...>` tables

Reply:
Ah okay. That causes an extra join; but no problem.

Also I'm not sure what we'll do once we start translating. We'll probably drop the names from the backend and just go by codes.

Reply:
>Ah okay. That causes an extra join; but no problem.

Yes, but I needed a join for alphabetization anyway (and this is the only way to do it, even with two joins)

>Also I'm not sure what we'll do once we start translating. We'll probably drop the names from the backend and just go by codes.

Was wondering that. Alas.

Reply:
The thing is that the frontend can do this easily since we don't paginate these, so the frontend can sort a localized list (not just names, but sorting is also different in different locales!)

Reply:
True. Easy to remove if/when they ever implement.